src/styles/blocks/place.scss

```diff
@@ -0,0 +1,50 @@
+.place {
+  &__photo-container {
+    position: relative;
+    box-sizing: border-box;
+    width: 100%;
+    max-height: 256px;
+    overflow: hidden;
+    transition: transform $transition-duration;
+
+    &:hover {
+      transform: scale(1.1);
+      z-index: 1;
+    }
```

Comment:
Scale is too big, pictures overlap on each other on hover https://prnt.sc/1ttsymf